torch/lib/THPP/tensors/generic/THTensor.cpp

```diff
@@ -113,7 +122,7 @@ auto THTensor<real>::resize(THLongStorage *size,
 
 template<>
 auto THTensor<real>::resizeAs(const Tensor& src) -> THTensor& {
-  THTensor_(resizeAs)(tensor, dynamic_cast<const THTensor<real>&>(src).tensor);
```

Comment:
My concern is that `resizeAs` doesn't have a `const` qualifier, so how can we possible pass in a `.tensor` of a const object?